src/axom/quest/examples/quest_marching_cubes_example.cpp

```diff
@@ -90,30 +90,12 @@ struct Input
 
   bool checkResults {false};
 
-  quest::MarchingCubesRuntimePolicy policy {
-    quest::MarchingCubesRuntimePolicy::seq};
+  axom::core::runtime_policy::Policy policy {
```

Comment:
Minor: This file might benefit from a type alias for `axom::Core::runtime_policy::Policy`

Reply:
Done.